_posts/2024-05-14-pyos-guide-to-pyconus-2024.md

```diff
@@ -0,0 +1,104 @@
+
+### <i class="fa-solid fa-users-between-lines"></i> 1:45--5:45 PM: Packaging Summit
+*David L. Lawrence Convention Center*\
+The goal of the [Packaging Summit](https://us.pycon.org/2024/events/packaging-summit/) is to try to take advantage of the fact that, at PyCon, we can get a high concentration of these stakeholders in one room at the same time. This allows us to sync up on current and future best practices and to quickly come to agreements that would take months or even years over higher-latency media of discussion (e.g. mailing lists, forums, issue tickets).
+
+### <i class="fa-solid fa-comment"></i> 2:30--3:00 PM: NetworkX is Fast Now: Graph Analytics Unleashed
+*Hall C*\
+**[Mridul Seth](https://github.com/MridulS) & [Erik Welch](https://github.com/eriknw)**\
+Have you ever wondered how to find connections in your data and to gain insights from them? Come discover how NetworkX makes this easy (and fast!).
+
+This talk is broadly divided into two parts. First we will talk about the power of graph analytics and how you can use tools like NetworkX to extract information from your data, and then we will talk about how we made the machinery behind NetworkX work with heterogeneous backends like GraphBLAS (CPU optimized) and cuGraph (GPU optimized).
+
+### <i class="fa-solid fa-comment"></i> 2:30--3:00 PM: Eternal sunshine of the spotless development environment
+*Room 301--305*\
+**[Sarah Kaiser](https://github.com/crazy4pi314)**\
+In this talk, Sarah will briefly cover why setting up container infrastructure, which can be useful for isolating your project environments and dig into how you can extend that with Dev Containers to configure a complete development experience using VS Code. She'll also look at two common OSS project situations, onboarding and workshops, to see how workflows for using Dev Containers and other supporting tools make things easier. No container experience required, and while a brief familiarity with VS Code is helpful, it's not necessary.
+
+## <i class="fa-regular fa-calendar"></i> Sunday, May 18th
+### <i class="fa-solid fa-users-between-lines"></i> 10:00--3:15 PM: Documentation Summit
+The [Documentation Summit](https://us.pycon.org/2024/events/hatchery/docs-summit/) will be a full-day summit including talks and panel sessions inviting leaders in documentation to share their experience in how to make good documentation, discussion about documentation tools such as sphinx, mkdocs, themes etc, what are the common mistakes and how to avoid them.
+
+And at 1:00 PM you can catch [Carol Willing](https://github.com/willingc) and [Mr. Batchelder](https://github.com/nedbat) giving an update from the Python Docs Editorial Board!
```

Comment:
pre-commit can be so tricky. i'd just fix this locally and use --no-verify to get it online! then you can add the codespell ignore stuff either in this PR or in a separate one. whatever you prefer :) but let's call Ned - Ned. 

this reminds me of when i read the times or WSJ - they always refer to people like mr musk and it always makes me chuckle. i don't know why however . 😆

Reply:
yes! it's so formal, and it seems even doubly so because it's in the middle of a blog post where everyone else is referred to by their first name.